src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -3938,6 +3939,13 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
+[node name="IslandError" type="AcceptDialog" parent="MicrobeEditorGUI/CellEditor"]
```

Comment:
I've found that it doesn't work super good with the pause menu, as you can open it with ESC, but then you can't actually click anything.
Optimal would be if it could be kept open but without messing with the pause menu.

Reply:
Oh yeah, that could be a problem, but I did made the negative atp popup exclusive though so hmm, I wonder if the pause menu could be made on top of popups. Anyway until there is any workaround to that I think it should just be disabled then.

Reply:
Do you mean the pause menu should be disabled? I did add that Tutorial exclusive popup state passing to it. Extending that to everything seems a bit troublesome.
If it's hard to make a popup that stays without exclusive on, this might be something to design a better solution later.

Reply:
I mean the popup's exclusive property, not the pause menu.

Reply:
Okay, I can agree with that. The popup reopening on pressing the finish button again should be good enough.